anaconda_project/client.py

```diff
@@ -145,6 +160,30 @@ def upload(self, project_info, archive_filename, uploaded_basename, private):
 
         return res.json()
 
+    def download(self, project, project_dir=None, parent_dir=None):
+        """Download project archive and extract."""
+        owner, project_name = project.split('/')
```

Comment:
It seems like we should support the case when the owner is not supplied, by defaulting to the logged-in user. Thoughts? After all, you do already have `-u USERNAME` as an option

Reply:
done